hpx/util/logging/format/formatter/high_precision_time.hpp

```diff
@@ -105,7 +104,8 @@ template<class convert = do_convert_format::prepend> struct high_precision_time_
         // fall back to non-thread-safe version on other platforms
         std::tm local_tm;
         {
-            std::unique_lock<hpx::lcos::local::spinlock> ul(gmtime_call_mutex);
+            static hpx::lcos::local::spinlock mutex;
+            std::unique_lock<hpx::lcos::local::spinlock> ul(mutex);
```

Comment:
This looks fine; note that this code is surrounded in the same #ifdef/#endif block as the above diff, although the #ifdef/#endif aren't visible in this patch..